docs/source/installation.rst

```diff
@@ -257,7 +211,7 @@ Virtual environments
 
 Two good options for virtual environment managers are:
 
-* `conda <https://uoa-eresearch.github.io/eresearch-cookbook/recipe/2014/11/20/conda/>`_ (many sktime community members us this)
+* `conda <https://uoa-eresearch.github.io/eresearch-cookbook/recipe/2014/11/20/conda/>`_ (beginner friendly, but may incur license fees for commercial use if using a commercial distribution).
```

Comment:
This link is quite old. May be link an official link, e.g. https://docs.conda.io/projects/conda/en/23.3.x/user-guide/getting-started.html?

Reply:
yes, it is indeed odd - I have not changed it though, it has been there since a while

Reply:
I updated both links now, they link to the official docs.